src/deploy/constants.ts

```diff
@@ -28,4 +28,7 @@ export const EXECUTOR_ROLE = ethers.solidityPackedKeccak256(
 
 export const ResolverTypes = {
```

Comment:
this would be better as an enum, then it would resolve the "TODO" comment you have below as these are just given values `0`, `1`, `2` etc. unless you specifically assign them something.

```solidity
export enum ResolverTypes {
  ADDRESS,
  STRING
}
```